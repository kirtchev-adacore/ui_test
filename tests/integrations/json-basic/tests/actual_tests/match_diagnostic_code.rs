fn main() {
    let _x: i32 = 0u32; //~ E0308
}
