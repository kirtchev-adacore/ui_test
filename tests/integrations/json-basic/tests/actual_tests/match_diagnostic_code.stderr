error[E0308]: mismatched types
 --> tests/actual_tests/match_diagnostic_code.rs:2:19
  |
2 |     let _x: i32 = 0u32;
  |             ---   ^^^^ expected `i32`, found `u32`
  |             |
  |             expected due to this
  |
help: change the type of the numeric literal from `u32` to `i32`
  |
2 |     let _x: i32 = 0i32;
  |                    ~~~

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.
